src/GP/GeneticOperations.java

```diff
@@ -0,0 +1,4 @@
+package GP;

```

Comment:
Hi. I wonder what is this file for?

Reply:
@bunverdenz read the message I sent to group yesterday